app/src/main/java/com/app/calculator/model/OperationModel.java

```diff
@@ -11,6 +11,9 @@ public double getOperation(String operator){
             case "+":
```

Comment:
the method should be named getOperationResult() you don't need the double result class attr if you calculate dynamically here